lib/prawn/grid.rb

```diff
@@ -145,12 +145,12 @@ def subdivide(total, num, gutter)
 
       def apply_gutter(options)
         if options.key?(:gutter)
-          @gutter = Float(options[:gutter])
+          @gutter = Float(options[:gutter] || 0.0)
```

Comment:
```suggestion
          @gutter = Float(options.fetch(:gutter, 0.0))
```

Reply:
Even better in this way, but I have found already this style in another part of the code, that's why I have used that.
We should change maybe also [there](https://github.com/prawnpdf/prawn/blob/ce374035905ef7691ddfa2dc015f4ea734d1f737/lib/prawn/document/bounding_box.rb#L534)

Reply:
I appreciate deference to existing code style but given that the project is relatively old I'd not pay very much attention to it.

I like `fetch`.